EpsilonLights/Src/Lights.c

```diff
@@ -225,23 +239,28 @@ void HAL_CAN_RxCpltCallback(CAN_HandleTypeDef* hcan)
     CanRxMsgTypeDef* msg = hcan->pRxMsg;
     HAL_GPIO_TogglePin(LED_RED_GPIO_Port, LED_RED_Pin);
 
-    if (msg->StdId == LIGHTS_INPUT_STDID)
+    if (msg->StdId == LIGHTS_INPUT_STDID && msg->DLC == 1)
     {
         lightsInputs = msg->Data[0];
     }
-    else if (msg->StdId == DRIVERS_INPUTS_STDID)
```

Comment:
In the future, don't mess with the order of the else ifs if you don't need to, this has caused an unnecessarily complicated diff, when this diff in `HAL_CAN_RxCpltCallback()` could've been a lot less verbose.